### DISCUSSION TITLE
Runtime Error: generator raised StopIteration

### DISCUSSION THREAD

**Discussion by chiakisantiago** [talmolab/sleap#972] [Help!]
After doing a single_instance model training, I wanted to run inference on suggested frames that SLEAP had given. After inputting code into Colab, I received a Runtime error: generator raised StopIteration. I'm not sure how to bypass this and still make a predictions.slp
<img width="478" alt="Screen Shot 2022-09-28 at 4 07 15 PM" src="https://user-images.githubusercontent.com/64056415/192903853-3a5c6b94-0183-4b0c-880f-3c6e37f37614.png">
<img width="282" alt="Screen Shot 2022-09-28 at 4 07 43 PM" src="https://user-images.githubusercontent.com/64056415/192903893-6604288e-4dbd-4a35-b804-156a20308cad.png">

**Comment by roomrys** [comment]
Hi @santiaci325,

Just wondering, when you exported the training job package, did you have "Predict On" set to "suggested frames"? If you did not, then the suggested frames would not be included in the exported `.pkg.slp`.

![image](https://user-images.githubusercontent.com/38435167/192910357-eb602f63-8cc5-410e-8792-66723797e97b.png)
**Fig 1:** To include labeling suggestions in the exported `.pkg.slp`, "Predict On" should be set to "suggested frames".

Let us know if that fixes things; otherwise, it might be a different issue.

Thanks,
Liezl
